data/json/monsters/reptile_amphibian.json

```diff
@@ -391,6 +392,47 @@
     "special_attacks": [ { "type": "leap", "cooldown": 10, "move_cost": 0, "max_range": 10, "min_consider_range": 2 } ],
     "anger_triggers": [ "STALK", "PLAYER_WEAK", "PLAYER_CLOSE" ],
     "flags": [ "SEES", "SMELLS", "HEARS", "SWIMS" ],
-    "armor": { "bash": 4 }
+    "armor": { "bash": 4 },
+    "upgrades": { "half_life": 65, "into": "mon_toad_beak" }
+  },
+  {
+    "id": "mon_toad_beak",
+    "type": "MONSTER",
+    "name": { "str": "great beaked toad" },
+    "description": "This massive toad's jawbones have grown out from its lips into a powerful beak, large enough to snap a bicycle in two. A pale fluid oozes from the creatures face and mouth. The swollen lumps behind its large eyes hint at this fluid's toxic nature.",
+    "default_faction": "frog",
+    "bodytype": "frog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "AMPHIBIAN" ],
+    "volume": "150 L",
+    "weight": "200 kg",
+    "hp": 200,
+    "speed": 80,
+    "symbol": "B",
+    "color": "brown",
+    "aggression": 10,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 2,
+    "melee_dice_sides": 12,
+    "melee_damage": [ { "damage_type": "cut", "amount": 2, "armor_multiplier": 0.5 } ],
+    "dodge": 1,
+    "harvest": "mutant_animal_large_noskin",
+    "path_settings": { "max_dist": 5 },
+    "special_attacks": [
+      {
+        "id": "bite_grab",
+        "cooldown": 10,
+        "accuracy": 6,
+        "damage_max_instance": [
+          { "damage_type": "cut", "amount": 16, "armor_multiplier": 0.3 },
+          { "damage_type": "bash", "amount": 8, "armor_multiplier": 0.3 }
+        ]
+      },
+      { "type": "leap", "cooldown": 10, "move_cost": 0, "max_range": 10, "min_consider_range": 2 }
+    ],
+    "anger_triggers": [ "STALK", "PLAYER_WEAK", "PLAYER_CLOSE" ],
+    "flags": [ "SEES", "SMELLS", "HEARS", "SWIMS", "GRABS", "VENOM" ],
```

Comment:
`VENOM` is very underpowered, and is applied to every attack by the critter. Use one of the JSON venom effects on the bite itself.